src/components/Index/ContributorsSection.tsx

```diff
@@ -4,112 +4,96 @@ import {
-            Past Content Authors:
-          </p>
-          <ul className="text-gray-500 dark:text-gray-400 list-disc list-inside space-y-2">
-            {pastContentAuthors.map(author => (
-              <li key={author.name}>
-                <a
-                  href={`https://github.com/${author.githubUsername}`}
-                  target="_blank"
-                  rel="noreferrer"
-                  className="text-blue-600 dark:text-blue-400"
-                >
-                  {author.name}
-                </a>
-              </li>
-            ))}
-          </ul>
+              </a>{' '}
+              If you're in the United States, you can <br></br>
+              earn volunteer hours too!
+            </p>
+            <h1 className="mt-2 mb-8 text-3xl text-center leading-8 font-extrabold tracking-tight text-gray-900 dark:text-gray-100 sm:text-xl sm:leading-10">
+              Current Content Authors:
+            </h1>
+            <ul className="mx-auto grid grid-cols-2 gap-x-4 gap-y-8 sm:grid-cols-4 md:gap-x-6 lg:max-w-5xl lg:gap-x-8 lg:gap-y-12 xl:grid-cols-6">
+              {contentAuthors.map(author => (
+                <li key={author.name}>
+                  <div className="space-y-4">
+                    <img
+                      className="mx-auto h-20 w-20 rounded-full lg:w-24 lg:h-24"
+                      src="https://i5.walmartimages.com/asr/be185d13-7b20-4595-bcdd-23bbe85f0074_1.d8c962e9921cd7f824d40d2f70adec30.jpeg?odnWidth=612&odnHeight=612&odnBg=ffffff"
+                      alt=""
+                    />
+                    <div className="space-y-2">
+                      <div className="text-xs font-medium lg:text-sm">
+                        <a
+                          href={`https://github.com/${author.githubUsername}`}
+                          target="_blank"
+                          rel="noreferrer"
+                          className="text-blue-600 dark:text-blue-400"
+                        >
+                          <h3>{author.name}</h3>
+                        </a>{' '}
+                      </div>
+                      <p className="text-gray-500 dark:text-gray-400 list-disc list-inside text-xs space-y-2 mb-6">
+                        Since {author.sinceDate}
+                      </p>
+                    </div>
+                  </div>
+                </li>
+              ))}
+            </ul>
+            <h1 className="mt-2 mb-8 text-3xl text-center leading-8 font-extrabold tracking-tight text-gray-900 dark:text-gray-100 sm:text-xl sm:leading-10">
+              Past Content Authors:
+            </h1>
+            <ul className="mx-auto grid grid-cols-2 gap-x-4 gap-y-8 sm:grid-cols-4 md:gap-x-6 lg:max-w-5xl lg:gap-x-8 lg:gap-y-12 xl:grid-cols-6">
+              {pastContentAuthors.map(author => (
+                <li key={author.name}>
+                  <div className="space-y-4">
+                    <img
```

Comment:
is there a special component you need to use to lazyload and optimize the image? i'm not familiar w/ gatsby so idk

Reply:
it was in the template 🤷‍♂️

Reply:
i mean the template is for general use, but there might be something you need for gatsby? @thecodingwizard

Reply:
oh yes there is. use gatsby image https://www.gatsbyjs.com/plugins/gatsby-image/ (I don't remember if we use the outdated gatsby image or the newer gatsby image)